services/clusters-keeper/tests/unit/test_utils_clusters.py

```diff
@@ -140,6 +141,33 @@ def test_create_startup_script(
     )
 
 
+def test_create_startup_script_script_size_below_16kb(
```

Comment:
this should be a pre-commit hook in my mind, I can assist wrapping it if needed :--)

Reply:
as explained, the pre-commit hook is not possible because this is only one side of the issue. the EC2_ALLOWED_TYPES are factored in and not available here.